paper_sections/01-intro.Rmd

```diff
@@ -6,7 +6,7 @@
 ```
 Two of the most popular and powerful modelling techniques currently in use by ecologists are generalized additive models [GAMs; @wood_generalized_2017] for modelling flexible regression functions, and generalized linear mixed models ["hierarchical generalized linear models" (HGLMs) or simply "hierarchical models"; @Bolker:2009cs; @gelman2013bayesian] for modelling between-group variability in regression relationships.
 
-At first glance, GAMs and HGLMs are very different tools used to solve different problems. GAMs are used to estimate smooth functional relationships between predictor variables and the response. HGLMs, on the other hand, are used to estimate linear relationships between predictor variables and response (although nonlinear relationships can also be modeled through quadratic terms or other transformations of the predictor variables), but impose a structure where predictors are organized into groups (often referred to as "blocks") and the relationships between predictor and response may vary across groups. Either the slope or intercept, or both, may be subject to grouping. A typical example of HGLM use might be to include site-specific effects in a model of population counts, or to model individual level heterogeneity in a study with repeated observations of multiple individuals.
+At first glance, GAMs and HGLMs are very different tools used to solve different problems. GAMs are used to estimate smooth functional relationships between predictor variables and the response. HGLMs, on the other hand, are used to estimate linear relationships between predictor variables and response <!-- removed for brevity -->, but impose a structure where predictors are organized into groups (often referred to as "blocks") and the relationships between predictor and response may vary across groups. Either the slope or intercept, or both, may be subject to grouping. A typical example of HGLM use might be to include site-specific effects in a model of population counts, or to model individual level heterogeneity in a study with repeated observations of multiple individuals.
```

Comment:
I agree, this is more succinct; however, one of the the reviewers had asked that we include a comment about possible nonlinear models in HGLMs, so I'll add the parenthetical back in.